code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1455,6 +1455,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 						var/datum/antagonist/rev/rev = H.mind.has_antag_datum(/datum/antagonist/rev)
 						if(rev)
 							rev.remove_revolutionary(FALSE, user)
+						var/datum/antagonist/hivevessel/woke = H.is_wokevessel()
+						if(woke && woke.one_mind)
+							woke.one_mind.remove_member(target.mind)
+							target.mind.remove_antag_datum(/datum/antagonist/hivevessel)
```

Comment:
``` suggestion
							H.mind.remove_antag_datum(/datum/antagonist/hivevessel)

Reply:
i am genuinely ashamed of screwing up a four line change